js/farms.js

```diff
@@ -63,21 +63,47 @@ function handleWebsiteJson_(data) {
 
   set_mgt_zones.clear()
   mgt_zones.length = 0
+
+  var bounds = [];
```

Comment:
Can we stick with 'let' as much as possible rather than var. For this and line 85,...my liege

Reply:
let there be let